pkg/cloudprovider/providers/gce/gce_addresses_fakes.go

```diff
@@ -19,12 +19,14 @@ package gce
 import (
 	"fmt"
 	"net/http"
+	"sync"
 
 	compute "google.golang.org/api/compute/v1"
 	"google.golang.org/api/googleapi"
 )
 
 type FakeCloudAddressService struct {
+	sync.RWMutex
```

Comment:
What's the point of the lock if each unit test is instantiating its own instance?